package proto_ff;

import "yd_fieldoptions.proto";


message duplicateduplicate
{
	optional int64 duplicateID = 1[(yd_fieldoptions.field_cname) = "副本ID"];
	optional string duplicateName = 2[(yd_fieldoptions.field_cname) = "副本名称", (yd_fieldoptions.field_bufsize) = 32];
	optional string duplicateTitlePic = 3[(yd_fieldoptions.field_cname) = "副本标题图片", (yd_fieldoptions.field_bufsize) = 32];
	optional int64 groupID = 4[(yd_fieldoptions.field_cname) = "所属副本组ID"];
	optional int32 duplicateType = 5[(yd_fieldoptions.field_cname) = "副本内容类型"];
	optional int32 playType = 6[(yd_fieldoptions.field_cname) = "副本玩法类型"];
	optional int64 sceneResource = 7[(yd_fieldoptions.field_cname) = "场景ID"];
	optional int32 levelLimit = 8[(yd_fieldoptions.field_cname) = "等级要求下限"];
	optional int64 linkDuplicateRequest = 9[(yd_fieldoptions.field_cname) = "前置副本要求"];
	optional int32 isAutoFight = 10[(yd_fieldoptions.field_cname) = "是否允许自动战斗"];
	optional string duplicateConditionType = 11[(yd_fieldoptions.field_cname) = "副本胜利条件类型", (yd_fieldoptions.field_bufsize) = 32];
	optional int32 reviveMaxTimes = 12[(yd_fieldoptions.field_cname) = "复活次数上限"];
	optional int64 dropNormalMonster = 13[(yd_fieldoptions.field_cname) = "普通怪物掉落"];
	optional int64 dropEliteMonster = 14[(yd_fieldoptions.field_cname) = "精英怪物掉落"];
	optional int64 dropBossMonster = 15[(yd_fieldoptions.field_cname) = "首领怪物掉落"];
	optional string sceneAIScript = 16[(yd_fieldoptions.field_cname) = "场景AI脚本", (yd_fieldoptions.field_bufsize) = 32];
	optional string bossDesc = 17[(yd_fieldoptions.field_cname) = "首领说明序列", (yd_fieldoptions.field_bufsize) = 32];
	optional int32 TeamObjectID = 18[(yd_fieldoptions.field_cname) = "组队目标ID"];
	optional string gradeCondition = 19[(yd_fieldoptions.field_cname) = "星级时间", (yd_fieldoptions.field_bufsize) = 32];
	optional int64 sweepRewardBoxID = 20[(yd_fieldoptions.field_cname) = "扫荡奖励BOXID"];
	optional int32 sweepRewardExp = 21[(yd_fieldoptions.field_cname) = "扫荡经验奖励"];
	optional string trackpath = 22[(yd_fieldoptions.field_cname) = "副本寻路点", (yd_fieldoptions.field_bufsize) = 119];
	optional string pathOfmonster = 23[(yd_fieldoptions.field_cname) = "副本寻路点对应怪物", (yd_fieldoptions.field_bufsize) = 121];
	optional int32 isPatrol = 24[(yd_fieldoptions.field_cname) = "寻路前是否巡敌"];
	optional int32 sweepType = 25[(yd_fieldoptions.field_cname) = "扫荡类型"];
}


message Sheet_duplicateduplicate
{
	repeated duplicateduplicate duplicateduplicate_List = 1[(yd_fieldoptions.field_arysize)=911];
}

message duplicateconstant
{
	optional int64 constantid = 1[(yd_fieldoptions.field_cname) = "配置id"];
	optional int64 constantdata = 2[(yd_fieldoptions.field_cname) = "数据"];
	optional string stringdata = 3[(yd_fieldoptions.field_cname) = "字符串数据", (yd_fieldoptions.field_bufsize) = 32];
}


message Sheet_duplicateconstant
{
	repeated duplicateconstant duplicateconstant_List = 1[(yd_fieldoptions.field_arysize)=106];
}

message duplicateendlessMaze
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 level = 2[(yd_fieldoptions.field_cname) = "等级下限"];
	optional string duplicateID = 3[(yd_fieldoptions.field_cname) = "副本id", (yd_fieldoptions.field_bufsize) = 224];
	optional int32 CreateRate = 4[(yd_fieldoptions.field_cname) = "分配到新副本的概率"];
	optional int64 rewardID = 5[(yd_fieldoptions.field_cname) = "参与奖励id"];
}


message Sheet_duplicateendlessMaze
{
	repeated duplicateendlessMaze duplicateendlessMaze_List = 1[(yd_fieldoptions.field_arysize)=117];
}

message duplicatetowerReward
{
	optional int64 floorMin = 1[(yd_fieldoptions.field_cname) = "层数下限"];
	optional int64 floormax = 2[(yd_fieldoptions.field_cname) = "层数上限"];
	optional int64 BOXID = 3[(yd_fieldoptions.field_cname) = "奖励BOX"];
	optional int64 itemId = 4[(yd_fieldoptions.field_cname) = "纹章经验"];
	optional int32 itemNum = 5[(yd_fieldoptions.field_cname) = "纹章经验数量"];
}


message Sheet_duplicatetowerReward
{
	repeated duplicatetowerReward duplicatetowerReward_List = 1[(yd_fieldoptions.field_arysize)=145];
}

message duplicatedungeonRewardrewardDesc
{
	optional int32 ItemNum = 1[(yd_fieldoptions.field_cname) = "道具数量"];
	optional int64 Item = 2[(yd_fieldoptions.field_cname) = "道具"];
	optional int32 Num = 3[(yd_fieldoptions.field_cname) = "序号"];
}

message duplicatedungeonReward
{
	optional int32 levelID = 1[(yd_fieldoptions.field_cname) = "层数"];
	optional int32 rewardTime = 2[(yd_fieldoptions.field_cname) = "领取次数上限"];
	optional int32 passReceiveTime = 3[(yd_fieldoptions.field_cname) = "通关获得次数"];
	repeated duplicatedungeonRewardrewardDesc reward = 4[(yd_fieldoptions.field_cname) = "奖励", (yd_fieldoptions.field_arysize) = 10];
}


message Sheet_duplicatedungeonReward
{
	repeated duplicatedungeonReward duplicatedungeonReward_List = 1[(yd_fieldoptions.field_arysize)=304];
}

message duplicategroup
{
	optional int64 id = 1[(yd_fieldoptions.field_cname) = "副本组ID"];
	optional string name = 2[(yd_fieldoptions.field_cname) = "副本组名", (yd_fieldoptions.field_bufsize) = 32];
	optional string duplicateTitlePic = 3[(yd_fieldoptions.field_cname) = "副本标题图片", (yd_fieldoptions.field_bufsize) = 32];
	optional int32 unlockLevel = 4[(yd_fieldoptions.field_cname) = "解锁等级"];
	optional int32 belongToType = 5[(yd_fieldoptions.field_cname) = "所属类型"];
	optional int32 viptype = 6[(yd_fieldoptions.field_cname) = "VIP类型"];
	optional int32 dynamiclevel = 7[(yd_fieldoptions.field_cname) = "是否动态等级"];
	optional string timesRes = 8[(yd_fieldoptions.field_cname) = "次数ICON资源", (yd_fieldoptions.field_bufsize) = 32];
	optional int32 groupTimesType = 9[(yd_fieldoptions.field_cname) = "组次数类型"];
	optional int32 groupTimes = 10[(yd_fieldoptions.field_cname) = "次数上限值"];
	optional int32 refreshType = 11[(yd_fieldoptions.field_cname) = "次数上限刷新类型"];
	optional int64 enterItemID = 12[(yd_fieldoptions.field_cname) = "副本进入道具"];
	optional int32 reliveTpye = 13[(yd_fieldoptions.field_cname) = "复活类型"];
	optional string sweepCondition = 14[(yd_fieldoptions.field_cname) = "扫荡条件", (yd_fieldoptions.field_bufsize) = 32];
	optional int32 buyEnterTimesCostType = 15[(yd_fieldoptions.field_cname) = "购买次数消耗货币"];
	optional int32 buyEnterTimesCost = 16[(yd_fieldoptions.field_cname) = "购买进入次数消耗"];
	optional int32 countDown = 17[(yd_fieldoptions.field_cname) = "倒计时时间"];
	optional int32 pathfindingtype = 18[(yd_fieldoptions.field_cname) = "追踪类型"];
}


message Sheet_duplicategroup
{
	repeated duplicategroup duplicategroup_List = 1[(yd_fieldoptions.field_arysize)=133];
}

message duplicatesweepmonsterNum
{
	optional int32 ID = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 sweepType = 2[(yd_fieldoptions.field_cname) = "扫荡类型"];
	optional int32 minLevel = 3[(yd_fieldoptions.field_cname) = "玩家等极下限"];
	optional int32 maxLevel = 4[(yd_fieldoptions.field_cname) = "玩家等极上限"];
	optional int32 killNumber = 5[(yd_fieldoptions.field_cname) = "杀怪标准数量"];
	optional int32 monstervaluetype = 6[(yd_fieldoptions.field_cname) = "怪物数值类型"];
}


message Sheet_duplicatesweepmonsterNum
{
	repeated duplicatesweepmonsterNum duplicatesweepmonsterNum_List = 1[(yd_fieldoptions.field_arysize)=155];
}

message duplicateGuildbosstime
{
	optional int64 duplicateID = 1[(yd_fieldoptions.field_cname) = "副本ID"];
	optional int32 openCycle = 2[(yd_fieldoptions.field_cname) = "副本开启星期"];
	optional string openTime = 3[(yd_fieldoptions.field_cname) = "副本开启时间", (yd_fieldoptions.field_bufsize) = 32];
	optional int32 bossGroup = 4[(yd_fieldoptions.field_cname) = "公会boss组ID"];
	optional int32 relevance = 5[(yd_fieldoptions.field_cname) = "互斥标记"];
}


message Sheet_duplicateGuildbosstime
{
	repeated duplicateGuildbosstime duplicateGuildbosstime_List = 1[(yd_fieldoptions.field_arysize)=325];
}

message duplicateGuildbosskillDesc
{
	optional int32 small = 1[(yd_fieldoptions.field_cname) = "最小次数"];
	optional int32 large = 2[(yd_fieldoptions.field_cname) = "最大次数"];
	optional int32 weaken = 3[(yd_fieldoptions.field_cname) = "副本属性"];
}

message duplicateGuildboss
{
	optional int64 duplicateID = 1[(yd_fieldoptions.field_cname) = "副本ID"];
	optional int64 rewardId = 2[(yd_fieldoptions.field_cname) = "领取BOXID"];
	repeated duplicateGuildbosskillDesc kill = 3[(yd_fieldoptions.field_cname) = "击败", (yd_fieldoptions.field_arysize) = 6];
}


message Sheet_duplicateGuildboss
{
	repeated duplicateGuildboss duplicateGuildboss_List = 1[(yd_fieldoptions.field_arysize)=325];
}

message duplicateGuildDuplicate
{
	optional int32 level = 1[(yd_fieldoptions.field_cname) = "玩家等级"];
	optional int64 monsterExp = 2[(yd_fieldoptions.field_cname) = "小怪经验"];
	optional int64 bossExp = 3[(yd_fieldoptions.field_cname) = "boss经验"];
}


message Sheet_duplicateGuildDuplicate
{
	repeated duplicateGuildDuplicate duplicateGuildDuplicate_List = 1[(yd_fieldoptions.field_arysize)=1604];
}
